### DISCUSSION TITLE
How does the "Recent" profile section work?

### DISCUSSION THREAD

**Discussion by nikunaku** [ppy/osu#28181] [Q&A] [answered]
Whenever I check the "Recent" section, the only thing that show up are medals and an [osu!mania convert-score](https://osu.ppy.sh/scores/2781142344) (mania is not my main gamemode):
![recent'nt tab](https://github.com/ppy/osu/assets/166950173/89e0f421-23aa-4d4f-92ed-8484ccedb101)
This is weird, as I've read that entries include _beatmap scores that are rank ≥1k_, and I'm sure I've set many scores that are above that.

I've also tried to reproduce this by setting [three](https://osu.ppy.sh/scores/2834300263) [more](https://osu.ppy.sh/scores/2836868431) [scores](https://osu.ppy.sh/scores/2836929396) of similar ranking but it didn't work; osu!, osu!mania and osu!mania-convert respectively.

Profile: https://osu.ppy.sh/users/33795951

Does lazer do this differently or is this some kind of bug?

**Comment by bdach** [comment, marked as answer]
Tracked at https://github.com/ppy/osu-infrastructure/issues/27.

tl;dr: The rank events don't currently work for lazer scores.
